Support/Scripts/start-android-emulator

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import os
+import argparse
+import subprocess
+from typing import Dict, Any, Tuple
+import sys
+import copy
+
+
+def run_argument_parser() -> argparse.Namespace:
```

Comment:
Fancy typing. I like it.

Reply:
Nit: The name is verbose. I'd do something like `parse_args()` or `parse_arguments()`.